svm-builds/Cargo.toml

```
@@ -15,14 +15,14 @@ build = "build.rs"
 [build-dependencies]
 build_const = "0.2.2"
 svm = { package = "svm-rs", version = "0.2.22", path = "..", features = ["blocking"]}
-semver = { version = "1.0.4", default-features = false, features = ["std", "serde"] }
+semver = { version = "1.0.17", default-features = false, features = ["std", "serde"] }
```

I would prefer changing these to regular expressions, like in this case:
```suggestion
semver = { version = "^1.0.0", default-features = false, features = ["std", "serde"] }
```

---

And similarly in other places, so that we update the minor version in the PR and allow patches to be updated by `cargo update` in the future.

---

`^1.0.0` is the default strategy, which is the same as `1.0.0`: <https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements>

I can set patch version to `0` or remove it if that's what you want

---

We can use `~1.0` in the above case, and similarly elsewhere then.

---

Done in #81 since that PR will modify `Cargo.toml`s